data/json/recipes/recipe_food.json

```diff
@@ -1398,6 +1412,138 @@
     "tools": [ [ [ "water_boiling_heat", 3, "LIST" ] ] ],
     "components": [ [ [ "coffee_raw", 1 ], [ "coffee_syrup", 1 ] ], [ [ "water", 1 ], [ "water_clean", 1 ] ] ]
   },
+  {
+    "type": "recipe",
+    "result": "espresso",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_DRINKS",
+    "skill_used": "cooking",
+    "difficulty": 1,
+    "time": "10 m",
+    "autolearn": true,
+    "batch_time_factors": [ 80, 4 ],
+    "qualities": [ { "id": "BOIL", "level": 1 } ],
```

Comment:
This is not espresso, as a Seattleite I cannot let this stand.  At best this is Turkish coffee.